package.json

```diff
@@ -7,7 +7,7 @@
   },
   "files": [
     "dist",
-    "src/**/*",
+    "build",
```

Comment:
👍 the industry defacto approach is just to add compiled code to a dist or lib folders. Perhaps it's best to add there under `helpers` or `build` folder? E.g. `dist/build` or `dist/configs`?

In terms of breaking changes, it's entirely up to you how you communicate breaking changes and use SemVer. As Primitives is OSS, technically _any_ removal of files in a new version is a breaking change, and should be communicated as such. A 'contract' was made with users the moment you made it publicly available. SemVer is one way to comm that, but there's other ways too. If you remove, or rename it in future but don't want to create a `major` update because it's more experimental, you could manually call it out in the Changelog like this: https://github.com/primer/brand/releases/tag/%40primer%2Freact-brand%400.35.0. This might be sufficient.